feat: aggiunta capitolo community e networking (#195)
Il-Libro-Open-Source/book: docs/it/community-e-networking.md

# Community e Networking

Questo capitolo nasce per due motivi: Il primo è quello di sfatare il mito del dev asociale, il secondo è quello di spiegare come fare networking e perché è così importante in un settore come il nostro.

## Asocialità nel mondo Dev

Chi sviluppa software è spesso visto come una persona asociale, tendenzialmente nerd e propensa a passare le sue giornate senza interagire con nessuno. Eppure, il mondo dello sviluppo software è tra i più vivi da un punto di vista di networking e community. Come mai, quindi, questa percezione è così radicata?

### Perché si è propensi a pensare che i dev siano asociali?

Non possiamo negare che esistano sviluppatori o sviluppatrici che non amano interagire con gli altri o stare in mezzo alla gente e che preferiscono passare le loro giornate in solitudine. Questo è vero, ma sarebbe sbagliato pensare che questo sia un tratto distintivo della nostra professione. In ogni settore lavorativo esistono persone che incarnano queste caratteristiche, quindi non è corretto pensare che il dev sia asociale per natura.

> Detto questo, è vero che lo sviluppo software è un lavoro che richiede molta concentrazione e che spesso si lavora in solitudine, il che può portare a una certa tendenza all'isolamento. Soprattutto nei primi decenni di vita della nostra professione, quando non esistevano ancora i social network e le community online, era molto più facile per chi facesse questo lavoro isolarsi, il che ha portato alla nascita di una percezione così radicata che spesso anche chi fa parte del settore si stupisce di quante community esistano e di quanto sia facile fare networking.

Reviewer: Che ne direste di rielaborare? Potrebbe passare l'idea che si debba entrare in "monk mode" durante i primi dieci anni di carriera.

```suggestion
Detto questo, è vero che lo sviluppo software è un lavoro che richiede molta concentrazione e che spesso si lavora in solitudine, il che può portare a una certa tendenza all'isolamento. Agli albori dell'informatica, quando non esistevano ancora i social network e le community online, era molto più facile per chi facesse questo lavoro isolarsi, il che ha portato alla nascita di una percezione così radicata che spesso anche chi fa parte del settore si stupisce di quante community esistano e di quanto sia facile fare networking.
```

### Esistono Dev non asociali?

Come abbiamo visto, la percezione del dev asociale è radicata, ma non corretta. Lo sviluppo software è di per sé un lavoro che spesso richiede di isolarsi, ma la carriera di chi sviluppa software è molto più ricca di questo. Il networking e la partecipazione alle community sono due aspetti fondamentali per chiunque voglia intraprendere questa carriera, tanto che spesso sono considerati requisiti fondamentali per chi vuole fare carriera.

> Il ruolo del dev è cambiato molto negli ultimi anni e oggi è molto più vicino al business e alla comunicazione rispetto a quanto non fosse in passato, o perlomeno, lo è per chi vuole fare il salto di qualità. Questo significa che il dev deve essere in grado di comunicare con i colleghi, con i clienti e con i superiori, deve essere in grado di vendere le proprie idee e di convincere gli altri a seguirlo. Queste sono tutte competenze che si acquisiscono con l'esperienza, ma che possono essere affinate anche attraverso il networking e la partecipazione alle community.

Author: ```suggestion
Il ruolo del dev è cambiato molto negli ultimi anni e oggi è molto più vicino al business e alla comunicazione rispetto a quanto non fosse in passato, o perlomeno, lo è per chi vuole fare il salto di qualità. Questo significa che un/una dev deve essere in grado di comunicare con colleghi di altri reparti, con i clienti e con i superiori, deve essere in grado di vendere le proprie idee e di convincere gli altri a seguire le sue idee. Queste sono tutte competenze che si acquisiscono con l'esperienza, ma che possono essere affinate anche attraverso il networking e la partecipazione alle community.
```

Questi collettivi si incontrano online o offline per confrontarsi, imparare e crescere insieme. Gli argomenti sono tra i più disparati, ma spesso si tratta di argomenti tecnici, come un linguaggio di programmazione, un framework o una libreria.

### Perché partecipare alle community?

Consideriamo la partecipazione alle community fondamentale, ma non obbligatoria, per chiunque voglia intraprendere la carriera di sviluppatore. Le community come detto sono un luogo di crescita personale e professionale, permettono inoltre di fare networking e ampliare la propria cerchia di conoscenze e amicizie, le quali potrebbero aiutarvi ad aprirvi molte possibilità future.

Le community inoltre sono uno dei migliori modi per imparare qualcosa di nuovo, perché tendenzialmente in queste avviene uno scambio di conoscenze gratuite tra i vari membri! Quello che potreste apprendere un giorno, potreste insegnarlo ad altri!

Ovviamente tra i pro non possiamo non citare il divertimento, le community sono luoghi di ritrovo di persone con uguali interessi, quindi potrebbe essere più facile per una persona creare un legame con queste.

### Come partecipare alle community?

> Partecipare alle community è molto semplice. Esistono community online e offline, quindi possiamo scegliere quella che più ci si addice. Le community online sono molto più facili da raggiungere, ma quelle offline sono molto più ricche dal punto di vista del networking. Per non fare torti a nessuno, in questo capitolo non citeremo alcuna community in particolare, ma ci limiteremo a dire che molte di queste community sono raggiungibili tramite i social network, come Facebook, Twitter, Instagram, LinkedIn, Telegram, Discord, Slack, ecc. e molte delle persone più attive sui social fanno parte di diverse community, quindi spesso sarà sufficiente seguire queste per entrare a far parte del loro mondo.

Reviewer 2: Forse tra le piattaforme metterei anche Meetup (specificando che serve per le community offline)

Author: Makes sense.

### Come creare una community?

Interessi in comune, passione e costanza sono gli ingredienti fondamentali per creare una community. Se abbiamo un interesse in comune con altre persone, una passione che vogliamo condividere e la costanza di portare avanti un progetto, possiamo creare una community. Non esiste una ricetta magica per farlo, ma esistono alcune regole che possiamo seguire per aumentare le probabilità di successo.

> - **Sii inclusivo**: Le community sono fatte di persone e le persone sono fatte di differenze. Se vogliamo creare una community, dobbiamo essere inclusivi e accettare le differenze. Non possiamo pensare di creare una community se non siamo disposti ad accettare le differenze.

Reviewer: ```suggestion
- **Sii inclusivo**: Le community sono fatte di persone e ogni persona è diversa dall'altra. Se vogliamo creare una community, dobbiamo essere inclusivi e accettare le diversità. Non possiamo pensare di creare una community se non siamo disposti ad accettare questo.
```

- **Sii inclusivo**: Le community sono fatte di persone e le persone sono fatte di differenze. Se vogliamo creare una community, dobbiamo essere inclusivi e accettare le differenze. Non possiamo pensare di creare una community se non siamo disposti ad accettare le differenze.
- **Sii costante**: La costanza è fondamentale per creare una community. Se io stesso non partecipo attivamente alla vita del mio gruppo, perché dovrebbero farlo gli altri? Se vogliamo creare una community, dobbiamo essere i primi a partecipare attivamente alla vita del gruppo.
- **Trova un tema**: Le community nascono attorno a un tema, quindi dobbiamo trovare un tema che sia abbastanza ampio da poter attrarre un numero sufficiente di persone, ma abbastanza specifico da poter creare un gruppo di persone con interessi in comune.
- **Stabilisci delle regole**: Se vogliamo creare una community, dobbiamo stabilire delle regole che permettano alle persone di interagire in maniera civile e rispettosa. Questo aspetto risulta sempre più importante con l'aumentare del numero di persone che partecipano alla community.

> - **Non demoralizzarti**: Se ci tieni veramente alla tua community non demoralizzarti se questa non diventa popolare. Capiamo possa essere un bel traguardo, ma non è fondamentale! Anche una community più raccolta può risultare molto attiva e portare dei benefici ai suoi membri!

Reviewer 2: Qui aggiungerei tra i punti anche quello di "non chiudersi" ma collaborare con altre community e gruppi con lo scopo di aiutare loro a crescere e cercando di crescere tramite il loro aiuto

Author: Ci sta!

> - **Non demoralizzarti**: Se ci tieni veramente alla tua community non demoralizzarti se questa non diventa popolare. Capiamo che può essere bello, ma non è fondamentale! Anche la community più piccola, può magari essere più attiva di una enorme e può portare dei benefici ai suoi membri!

Author: ```suggestion
- **Non demoralizzarti**: Se ci tieni veramente alla tua community non demoralizzarti se questa non diventa popolare. Capiamo possa essere un bel traguardo, ma non è fondamentale! Anche una community più raccolta può risultare molto attiva e portare dei benefici ai suoi membri!
```

> - **Non chiudersi**: Il mondo delle community è fatto anche di collaborazione tra le stesse. Spesso, infatti, le community si aiutano a vicenda e si supportano, anche solo facendosi pubblicità a vicenda o scambiandosi consigli e risorse. Non chiudersi in se stessi e cercare di collaborare con altre community può portare a risultati inaspettati.

Reviewer: ```suggestion
- **Essere aperti**: Il mondo delle community è fatto anche di collaborazione tra le stesse. Spesso, infatti, le community si aiutano a vicenda e si supportano, anche solo facendosi pubblicità a vicenda o scambiandosi consigli e risorse. Non chiudersi in se stessi e cercare di collaborare con altre community può portare a risultati inaspettati.
```

Ovviamente, come precedentemente detto non esiste solo LinkedIn, ma anche partecipare a una community online affine può essere un ottimo modo per iniziare ad accrescere la propria rete di contatti.

Non esiste una formula magica da questo punto di vista, ma anche in questo caso, costanza e passione sono ingredienti fondamentali per intraprendere questo percorso.

### Come faccio networking offline?

Il networking offline è molto più difficile da intraprendere, ma è anche molto più efficace. Per fare networking offline è necessario partecipare a eventi, come meetup e conferenze, e interagire con le persone che partecipano a questi eventi.

Come potrai immaginare, in questo caso bisognerà essere il meno timidi possibile e non aver timore delle altre persone! Potresti cominciare ponendo delle domande senza pensare a se e quanto queste possano essere stupide, nessuno nasce con tutte le risposte pronte!

Come citato qualche paragrafo più in alto, non puoi mai sapere quanto una qualsiasi persona presente nella stanza possa influire sulla tua carriera, quindi cerca di interagire con tutti e cerca di conoscere il più possibile le persone che partecipano a questi eventi.

> Vuoi un esempio pratico del potere del networking? Questo capitolo fa parte di un libro scritto da una community di persone che condivide la passione per l'Open Source, e la prima stesura è stata fatta da due persone che si sono conosciute grazie ad un'amiciza comune, a sua volta dev, che ci ha presentati e ci ha permesso di conoscerci. A distanza di parecchi mesi da quel primo contatto, abbiamo deciso di scrivere questo capitolo insieme dopo esserci visti ad un paio di conferenze, a dimostrazione di quanto il networking sia imprevedibile quanto efficace.

Reviewer: Typo
```suggestion
Vuoi un esempio pratico del potere del networking? Questo capitolo fa parte di un libro scritto da una community di persone che condivide la passione per l'Open Source, e la prima stesura è stata fatta da due persone che si sono conosciute grazie ad un'amicizia in comune, a sua volta dev, che ci ha presentati e ci ha permesso di conoscerci. A distanza di parecchi mesi da quel primo contatto, abbiamo deciso di scrivere questo capitolo insieme dopo esserci visti ad un paio di conferenze, a dimostrazione di quanto il networking sia imprevedibile quanto efficace.
```

Detto questo, è vero che lo sviluppo software è un lavoro che richiede molta concentrazione e che spesso si lavora in solitudine, il che può portare a una certa tendenza all'isolamento. Soprattutto nei primi decenni di vita della nostra professione, quando non esistevano ancora i social network e le community online, era molto più facile per chi facesse questo lavoro isolarsi, il che ha portato alla nascita di una percezione così radicata che spesso anche chi fa parte del settore si stupisce di quante community esistano e di quanto sia facile fare networking.

### Dev asociale: non esiste

Come abbiamo visto, la percezione del dev asociale è radicata, ma non corretta. Lo sviluppo software è di per sé un lavoro che spesso richiede di isolarsi, ma la carriera di chi sviluppa software è molto più ricca di questo. Il networking e la partecipazione alle community sono due aspetti fondamentali per chiunque voglia intraprendere questa carriera, tanto che spesso sono considerati requisiti fondamentali per chi vuole fare carriera.

> Il ruolo del dev è cambiato molto negli ultimi anni e oggi è molto più vicino al business e alla comunicazione rispetto a quanto non fosse in passato, o perlomeno, lo è per chi vuole fare il salto di qualità. Questo significa che il dev deve essere in grado di comunicare con i colleghi, con i clienti e con i superiori, deve essere in grado di vendere le proprie idee e di convincere gli altri a seguirlo. Queste sono tutte competenze che si acquisiscono con l'esperienza, ma che possono essere affinate anche attraverso il networking e la partecipazione alle community.

Author: Fatto!

## Community

Le community sono un aspetto fondamentale per chiunque voglia intraprendere la carriera di sviluppatore. Sono un luogo dove poter imparare, confrontarsi e crescere, ma anche un luogo dove poter fare networking e conoscere persone che potrebbero aiutarci a crescere professionalmente.

### Cosa sono le community?

Le community sono gruppi di persone che condividono un interesse comune. Nel nostro caso, le community sono gruppi di persone che condividono l'interesse per lo sviluppo software. Queste persone si incontrano online o offline per confrontarsi, imparare e crescere insieme. Gli argomenti sono i più disparati, ma spesso si tratta di argomenti tecnici, come un linguaggio di programmazione, un framework o una libreria.

### Perché partecipare alle community?

> Partecipare alle community è fondamentale per chiunque voglia intraprendere la carriera di sviluppatore. Le community sono un luogo dove poter imparare, confrontarsi e crescere, ma anche un luogo dove poter fare networking e conoscere persone che potrebbero aiutarci a crescere professionalmente.

Reviewer 3: ```suggestion
Consideriamo la partecipazione alle community fondamentale, ma non obbligatoria, per chiunque voglia intraprendere la carriera di sviluppatore. Le community come detto sono un luogo di crescita personale e professionale, permettono inoltre di fare networking e ampliare la propria cerchia di conoscenze e amicizie, le quali potrebbero aiutarvi ad aprirvi molte possibilità future.

Le community inoltre sono uno dei migliori modi per imparare qualcosa di nuovo, perché tendenzialmente in queste avviene uno scambio di conoscenze gratuite tra i vari membri! Quello che potreste apprendere un giorno, potreste insegnarlo ad altri!

Ovviamente tra i pro non possiamo non citare il divertimento, le community sono luoghi di ritrovo di persone con uguali interessi, quindi potrebbe essere più facile per una persona creare un legame con queste.
```

> Spesso si dice che il modo migliore per imparare qualcosa è insegnarla agli altri e partecipare alle community ci permette di mettere in pratica le nostre conoscenze e di imparare cose nuove.

Reviewer 3: ```suggestion
```

### Come partecipare alle community?

> Partecipare alle community è molto semplice. Esistono community online e offline, quindi possiamo scegliere quella che più ci si addice. Le community online sono molto più facili da raggiungere, ma quelle offline sono molto più ricche dal punto di vista del networking. Per non fare torti a nessuno, in questo capitolo non citeremo alcuna community in particolare, ma ci limiteremo a dire che molte di queste community sono raggiungibili tramite i social network, come Facebook, Twitter, Instagram, LinkedIn, Telegram, Discord, Slack, ecc. e molte delle persone più attive sui social sono creatori e creatrici di community, quindi spesso sarà sufficiente seguire queste persone per entrare a far parte del loro mondo.

Reviewer 3: ```suggestion
Partecipare alle community è molto semplice. Esistono community online e offline, quindi possiamo scegliere quella che più ci si addice. Le community online sono molto più facili da raggiungere, ma quelle offline sono molto più ricche dal punto di vista del networking. Per non fare torti a nessuno, in questo capitolo non citeremo alcuna community in particolare, ma ci limiteremo a dire che molte di queste community sono raggiungibili tramite i social network, come Facebook, Twitter, Instagram, LinkedIn, Telegram, Discord, Slack, ecc. e molte delle persone più attive sui social fanno parte di diverse community, quindi spesso sarà sufficiente seguire queste per entrare a far parte del loro mondo.
```

- **Sii inclusivo**: Le community sono fatte di persone e le persone sono fatte di differenze. Se vogliamo creare una community, dobbiamo essere inclusivi e accettare le differenze. Non possiamo pensare di creare una community se non siamo disposti ad accettare le differenze.
- **Sii costante**: La costanza è fondamentale per creare una community. Se io stesso non partecipo attivamente alla vita del mio gruppo, perché dovrebbero farlo gli altri? Se vogliamo creare una community, dobbiamo essere i primi a partecipare attivamente alla vita del gruppo.
- **Trova un tema**: Le community nascono attorno a un tema, quindi dobbiamo trovare un tema che sia abbastanza ampio da poter attrarre un numero sufficiente di persone, ma abbastanza specifico da poter creare un gruppo di persone con interessi in comune.

> - **Stabilisci delle regole**: Se vogliamo creare una community, dobbiamo stabilire delle regole che permettano alle persone di interagire in maniera civile e rispettosa. Questo aspetto risulta sempre più importante con l'aumentare del numero di persone che partecipano alla community.

Reviewer 3: ```suggestion
- **Stabilisci delle regole**: Se vogliamo creare una community, dobbiamo stabilire delle regole che permettano alle persone di interagire in maniera civile e rispettosa. Questo aspetto risulta sempre più importante con l'aumentare del numero di persone che partecipano alla community.
- **Non demoralizzarti**: Se ci tieni veramente alla tua community non demoralizzarti se questa non diventa popolare. Capiamo che può essere bello, ma non è fondamentale! Anche la community più piccola, può magari essere più attiva di una enorme e può portare dei benefici ai suoi membri!
```

## Networking

> Il networking è un aspetto fondamentale per chiunque voglia intraprendere la carriera di sviluppatore. Per networking intendiamo la capacità di creare e mantenere relazioni con altre persone, in particolare con altre persone che lavorano nel nostro stesso settore o condividono i nostri stessi interessi.

Reviewer 3: ```suggestion
Con networking intendiamo la capacità di creare e mantenere relazioni con altre persone, in particolare con altre persone che lavorano nel nostro stesso settore o condividono i nostri stessi interessi.
```

### Networking all'inizio della carriera

> Il networking è fondamentale per chiunque voglia intraprendere la carriera di sviluppatore, ma è particolarmente importante per chi è all'inizio. Quando si è agli inizi, infatti, non si ha ancora un curriculum da poter mostrare e non si hanno ancora le competenze necessarie per poter essere assunti, quindi il networking è fondamentale per poter trovare le prime opportunità. Inoltre, quando si è agli inizi, si ha ancora la possibilità di scegliere il proprio percorso e il networking è fondamentale per poter conoscere le varie opportunità che il mondo dello sviluppo software offre.

Reviewer 3: ```suggestion
Il networking è importante per chiunque voglia intraprendere la carriera di sviluppatore, ma è particolarmente cruciale per chi è all'inizio. Non avendo ancora un curriculum da poter mostrare e non si hanno ancora le competenze necessarie per poter essere assunti, risultano fondamentali le connessioni che si creano per poter trovare le prime opportunità. Quando si è agli inizi, inoltre si ha ancora la possibilità di scegliere il proprio percorso e il networking è fondamentale per poter apprendere le varie opportunità che il mondo dello sviluppo software offre.
```

### Networking e carriera

> Non puoi mai sapere se la persona seduta accanto a te in un meetup o in una conferenza potrebbe essere il tuo prossimo datore di lavoro o il tuo prossimo cliente, o più semplicemente un futuro collega che ti aprirà le porte di un'azienda in cui vorresti lavorare.

Reviewer 3: ```suggestion
Partecipando a una community non puoi mai sapere se una determinata persona potrebbe diventare il tuo prossimo datore di lavoro o il tuo prossimo cliente, o più semplicemente un futuro collega che ti aprirà le porte di un'azienda in cui vorresti lavorare.
```

Author: Credo che l'aspetto più "diretto" del messaggio iniziale suoni meglio

> Fare networking può aprirti possibilità che nemmeno pensavi di avere e può aiutarti a crescere professionalmente in maniera esponenziale. Non è raro, infatti, che un dev venga assunto in un'azienda grazie a una conoscenza che lo ha raccomandato per un colloquio o che un dev noto internamente in un'azienda venga contattato dalla stessa per una posizione che non era stata ancora pubblicizzata, aumentando così le possibilità di essere assunto.

Reviewer 3: ```suggestion
Fare networking può aprirti possibilità che nemmeno pensavi di avere e può aiutarti a crescere professionalmente in maniera esponenziale. Non è raro, infatti, che un dev venga assunto in un'azienda grazie a una conoscenza che lo ha raccomandato per una posizione o che un dev noto internamente in un'azienda venga contattato dalla stessa per una posizione che non era stata ancora pubblicizzata, aumentando così le possibilità di essere assunto.

Sicuramente tutti questi pro sono molto importanti, ma una cosa che non andrebbe mai trascurata è la bellezza di poter conoscere gente nuova e creare nuove connessioni genuine e durature.
```

### Come faccio networking online?

> Negli ultimi anni (Decennio 2013-2023) il networking online è diventato sempre più importante e il grosso delle opportunità da questo punto di vista si trova su LinkedIn. Questo non significa che non si possa fare networking anche su altri social network, ma LinkedIn è sicuramente il social network più utilizzato per questo scopo.

Reviewer 3: ```suggestion
Negli ultimi anni (Decennio 2013-2023) il networking online è diventato sempre più importante e il grosso delle opportunità da questo punto di vista si trova su LinkedIn. Questo non significa che non si possa fare networking anche su altri social network, ma LinkedIn è sicuramente il social network più utilizzato per il nostro settore.
```

> Per fare networking su LinkedIn è necessario creare un profilo professionale, che sia in linea con il nostro settore di appartenenza e che sia aggiornato con le nostre esperienze e le nostre competenze. Una volta creato il profilo, sarà sufficiente iniziare a connettersi con altre persone, possibilmente con persone che lavorano nel nostro stesso settore o che condividono i nostri stessi interessi.

Reviewer 3: ```suggestion
Per fare networking su LinkedIn è necessario creare un profilo professionale, che sia in linea con il nostro settore di appartenenza e che sia aggiornato con le nostre esperienze e le nostre competenze. Una volta creato il profilo, sarà sufficiente iniziare a connettersi con altre persone, possibilmente con persone che potenzialmente condividano i nostri stessi interessi.
```

> Una volta connessi, sarà possibile interagire con queste persone, commentando i loro post o inviando loro messaggi privati. Questo ci permetterà di creare una relazione con queste persone e di farci conoscere da loro. Questo è il primo passo per fare networking su LinkedIn.

Reviewer 3: ```suggestion
Una volta connessi, sarà possibile interagire con queste persone, commentando i loro post o inviando loro messaggi privati. Questo ci permetterà di creare una relazione con loro e ci permetterà di farci conoscere. Questo è il primo passo per fare networking su LinkedIn.
```

> Non esiste una formula magica da questo punto di vista, ma anche in questo caso, costanza e passione sono ingredienti fondamentali per intraprendere questo percorso.

Reviewer 3: ```suggestion
Ovviamente, come precedentemente detto non esiste solo LinkedIn, ma anche partecipare a una community online affine può essere un ottimo modo per iniziare a crescere la propria rete di contatti.

Non esiste una formula magica da questo punto di vista, ma anche in questo caso, costanza e passione sono ingredienti fondamentali per intraprendere questo percorso.
```

### Come faccio networking offline?

> Il networking offline è molto più difficile da intraprendere, ma è anche molto più efficace. Per fare networking offline è necessario partecipare a eventi, come meetup e conferenze, e interagire con le persone che partecipano a questi eventi.

Reviewer 3: ```suggestion
Il networking offline è molto più difficile da intraprendere, ma è anche molto più efficace. Per fare networking offline è necessario partecipare a eventi, come meetup e conferenze, e interagire con le persone che partecipano a questi eventi.

Come potresti ben capire in questo caso bisognerà essere il meno timidi possibile e di non aver timore delle altre persone! Potete anche cominciare ponendo delle domande e non pensare se e quanto possano essere stupide, nessuno nasce con tutte le risposte pronte!
```

## Posso fare questo lavoro senza fare networking?

La risposta è, come sempre nel nostro settore, "dipende". Il networking è sicuramente un aspetto importante nella carriera dev, ma questo non significa che sia impossibile fare carriera senza fare networking.

Come ogni altro aspetto di questo lavoro, networking e community sono _opportunità_ che abbiamo e che possiamo usare come trampolino di lancio o come banco di prova, ma non sono _necessità_. Se non ci sentiamo a nostro agio nel partecipare alle community o nel fare networking, non dobbiamo forzarci, ma dobbiamo essere consapevoli che questo potrebbe limitare le nostre possibilità di crescita, o comunque renderle più difficili.

## Conclusioni

> In questo capitolo abbiamo visto come il networking e la partecipazione alle community siano due aspetti fondamentali per chiunque voglia intraprendere la carriera di sviluppatore. Sono due aspetti che possono aiutarci a crescere professionalmente e che possono aprirci opportunità che altrimenti non avremmo, ma è bene ricordare che non sono _necessità_, ma _opportunità_ che abbiamo e che possiamo sfruttare per crescere professionalmente.

Reviewer 3: ```suggestion
In questo capitolo abbiamo visto come il networking e la partecipazione alle community siano due aspetti fondamentali per chiunque voglia intraprendere la carriera di sviluppatore. Sono due aspetti che possono aiutarci a crescere professionalmente e che possono aprirci opportunità che altrimenti non avremmo, ma è bene ricordare che non sono _necessità_, ma _opportunità_ che abbiamo e che possiamo sfruttare per crescere sia personalmente che professionalmente.
```

### Come creare una community?

Interessi in comune, passione e costanza sono gli ingredienti fondamentali per creare una community. Se abbiamo un interesse in comune con altre persone, possiamo creare una community. Se abbiamo una passione che vogliamo condividere con altre persone, possiamo creare una community. Se abbiamo la costanza di portare avanti un progetto, possiamo creare una community. Non esiste una ricetta magica per creare una community, ma esistono alcune regole che possiamo seguire per aumentare le probabilità di successo.

> 

Reviewer 4: Forse un po’ troppe volte ripetuta (anche se capisci che é fatto intenzionalmente) la parola community

Author: Ciao @Mik23mik , ho rivisto il paragrafo, spero suoni meglio!
